test/templates/form/index.njk

```diff
@@ -208,6 +208,25 @@
 
     })
 }}
+{{ form_group(
```

Comment:
I'm not seeing that this is doing anything. It looks (and behaves) like the example above it.